public/scripts/group-chats.js

```diff
@@ -1020,6 +1024,48 @@ function activateListOrder(members) {
     return memberIds;
 }
 
+/**
+ * Activate group members based on the last message.
+ * @param {string[]} members List of member avatars
+ * @param {Object} lastMessage Last message
+ * @returns {number[]} List of character ids
+ */
+function activatePooledOrder(members, lastMessage) {
+    const activatedMembers = [];
```

Comment:
If there can only ever be one activated member in this function here, why make this an array?
Intentional code is always better, even though this doesn't hurt of course.